pkg/cli/adbtunnel.go

```diff
@@ -449,6 +492,26 @@ func (tc *TunnelController) handleControlCommand(conn net.Conn) {
 	}
 }
 
+// The returned status is only valid if the error is nil
+func sendStatusCmd(conn net.Conn) (ADBForwarderStatus, error) {
+	var msg ADBForwarderStatus
+	// unixseqpacket means we don't have to write or read in a loop, since all
```

Comment:
is this comment more relevant close to `conn, err := net.Dial("unixpacket", path)` in line 133

Reply:
I'm trying to explain why I don't write in a loop here, as is customary with `int write(buffer)`, so the comment is more relevant here. Rewrote it to try to make it clearer.